lib/error.js

```diff
@@ -41,6 +41,8 @@ var ERRNO = {
   BOUNCE_SOFT: 135,
   SERVER_BUSY: 201,
   FEATURE_NOT_ENABLED: 202,
+  EMAIL_EXISTS: 301,
+  EMAIL_DELETE_PRIMARY: 302,
```

Comment:
Why "3XX" here?  It seems reasonable to me to just keep incrementing the existing "1XX" error space, which would make these 136 and 137.